pkg/dataplane/util/ansibleee.go

```diff
@@ -140,6 +142,9 @@ func (a *EEJob) JobForOpenStackAnsibleEE(h *helper.Helper) (*batchv1.Job, error)
 	// Populate hash
 	hashes := make(map[string]string)
 
+	if a.PreserveJobs {
+		job.Spec.TTLSecondsAfterFinished = &TTLSecondsAfterFinishedValue
```

Comment:
Why set a value for TTLSecondsAfterFinished? It says "If the field is unset, this Job won't be cleaned up by the TTL controller after it finishes."

Reply:
Yeah, you're right. Maybe we just reverse this logic then and we only set it if `!PreserveJobs`.

Reply:
This is fine as a change, but how does this fix the issues we're seeing in CI? 

Also, please change/remove https://github.com/openstack-k8s-operators/openstack-operator/blob/main/pkg/dataplane/util/ansibleee.go#L49 as the default for bool is false.

Reply:
agree, it looks like PreserveJobs just gets hardcoded to True. we might as well drop it and just leave TTLSecondsAfterFinished unset

Reply:
Right, that makes sense. So to make sure we're clear, did you guys want to take another look at the updated change then. I'm just removing the `PreserveJobs` option from the `AEEJob` structure and leaving everything else the way it was since the defaults should be fine.

Reply:
in this version of the PR, a.PreserveJobs was hardcoded to True, so that's why the suggestion was to just drop it. But, to do so you also need to drop it from openstackdataplanedeployment_types.go and pkg/dataplane/util/ansible_execution.go.